engine/src/main/scala/cromwell/engine/io/IoActor.scala

```diff
@@ -179,6 +179,11 @@ object IoActor {
     case _ => false
   }
 
+  def is5xxServerError(failure: Throwable): Boolean = {
+    val serverErrorPattern = "^.*5\\d{2} \\w+(.*)?"
```

Comment:
This seems like a fairly loose regex, a 5 and two digits followed by a space and at least one word char, preceded by anything and followed by anything. No concern about false positives?

Reply:
The exact error class we're targeting (from @dinvlad) is
```
Bad output 'get_segs.count': Failed to read_int("count.txt") (reason 1 of 1): [Attempted 1 time(s)] - IOException: Could not read from gs://broad-epi-cromwell/workflows/ChipSeq/a4a8808f-32f3-4f3e-84f2-04fd7861c581/call-get_segs/shard-0/count.txt: 503 Service Unavailable Backend Error
```

I suppose we could be conservative by making it super specific and matching only on `503 Service Unavailable Backend Error`